### PR TITLE
Update Patterns with new var-length table and quantified relationship information

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

File: modules/ROOT/pages/patterns/reference.adoc [hunk 1]
```diff
@@ -1106,26 +1109,31 @@ For rules on valid relationship variable names, see xref:syntax/naming.adoc[Cyph
 [[variable-length-relationships-rules]]
 === Rules
 
-The following table shows variants of the variable-length quantifier syntax and their equivalent canonical xref:patterns/reference.adoc#quantifier[quantifier] form (the form used by xref:patterns/reference.adoc#quantified-path-pattern[quantified path patterns]):
+The following table shows variants of the variable-length quantifier syntax and their equivalent canonical xref:patterns/reference.adoc#quantifiers[quantifier] form (the form used by xref:patterns/reference.adoc#quantified-path-patterns[quantified path patterns]):
```
Comment (dogofbrian) [inline modules/ROOT/pages/patterns/reference.adoc]:
```suggestion
The following table shows variants of the variable-length quantifier syntax and their equivalent xref:patterns/reference.adoc#quantifiers[quantifier] form (the form used by xref:patterns/reference.adoc#quantified-path-patterns[quantified path patterns]):
```
As we are showing translations to + and * in new syntax, and these are not canonical forms, then we should not say that's what we're doing. I doubt the reader will be much concerned about what is canonical and what is shorthand.

File: modules/ROOT/pages/patterns/reference.adoc [hunk 2]
```diff
@@ -697,17 +697,20 @@ For example, the following quantified relationship:
...
+is equivalent to the following quantified path pattern:
 
 [source, role=noheader]
 ----
-() (()-[r]->()){m,n} ()
+(:A) (()-[r:R]->()){m,n} (:B)
 ----
 
-However, unlike a quantified path pattern, a quantified relationship must always have a node pattern on each side.
+With the expanded form of a quantified path pattern, it is possible to add predicates.
```
Comment (dogofbrian) [inline modules/ROOT/pages/patterns/reference.adoc]:
```suggestion
With the expanded form of a quantified path pattern, it is possible to add predicates inside.
```

**Review by dogofbrian — CHANGES_REQUESTED** [review]
Looks good! I just added a couple of small suggestions.

**Review by dogofbrian — APPROVED** [review]

**Pull request merged**